src/views/ProfileCreation/ProfilePicture.tsx

```diff
@@ -28,10 +31,39 @@ const ProfilePicture: React.FC = () => {
   const { library } = useWeb3React()
   const [isApproved, setIsApproved] = useState(false)
   const [isApproving, setIsApproving] = useState(false)
+  const [userProfileCreationNfts, setUserProfileCreationNfts] = useState(null)
   const { selectedNft, actions } = useContext(ProfileCreationContext)
+  const profileContract = useProfile(false)
 
   const { nfts, userNftsInitializationState } = useUserNfts()
-  useFetchUserNfts()
```

Comment:
We don’t need this, user nft list already fetched when profile creation page mounted or requested to fetch when minting operation is success